zwave_js_server/exceptions.py

```diff
@@ -63,6 +63,14 @@ class InvalidMessage(BaseZwaveJSServerError):
 class InvalidServerVersion(BaseZwaveJSServerError):
     """Exception raised when connected to server with incompatible version."""
 
+    def __init__(
+        self, current_version: int, required_version: int, message: str
```

Comment:
```suggestion
        self, current_version: str, required_version: int, message: str
```

Reply:
looked into it, current_version is the server version, required_version is the schema version. Changes incoming